cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -218,8 +218,8 @@ public void execute() {
             targetPath = sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.",
+                    "invalid Ballerina source path, it should either be a module name in a Ballerina project, a " +
+                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension or a .jar file.",
```

Comment:
I believe this should be
```suggestion
                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or a .jar file.",
```

Reply:
@praneesha I am a bit confused here. What would be the right way ?

Reply:
I changed it as:
> "invalid Ballerina source path, it should be a name of a module in a Ballerina project or a file with a '.bal' extension or an executable .jar file."

Reply:
Shall we reword it like this: "invalid Ballerina source path. It should either be a name of a module in a Ballerina project, a file with a '.bal' extension, or an executable .jar file."

Reply:
Fixed